.travis.yml

```diff
@@ -4,5 +4,8 @@ python:
   - "3.7"
 install:
   - pip install pylint
-# command to run tests
-script: pylint osbuild assemblers/* stages/*
+jobs:
+  include:
+    - stage: test
+      script: pylint osbuild assemblers/* stages/*
+    - script: sudo --preserve-env python3 -m osbuild --libdir . --output . samples/noop.json
```

Comment:
According to (how I understand) [the docs](https://docs.travis-ci.com/user/build-stages/#naming-your-build-stages), the default "stage" is "test", so that's redundant. I'd like to see some names in travis' job output though (otherwise it only says "Python 3.7" for each job. Could you try giving each job a name with `name: `?

Reply:
Done! Thanks!